.github/README.md

```diff
@@ -3,39 +3,28 @@
 
-Lawnicons is best used on Lawnchair 12.1. You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".
+Lawnicons is best used on [Lawnchair 12.1](https://github.com/LawnchairLauncher/lawnchair). You can enable themed icons on Lawnchair by going to `Home Settings > General > Icon Style > Themed Icons`, choosing "Home Screen" or "Home Screen and App Drawer".
 
-This project is mostly supported by our community. If you wish to contribute, see the [Contributing section](README.md#Contributing) below.
+This project is mostly supported by our community. If you wish to contribute, check out the Contributing section below.
 
 ## Download
 See the [Releases section](https://github.com/LawnchairLauncher/lawnicons/releases) to download the latest stable build of Lawnicons.
 
-For the development versions with new icons, go to [nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
+For the development versions with new icons, go to [the nightly.link](https://nightly.link/LawnchairLauncher/lawnicons/workflows/build_debug_apk/develop/Debug%20APK).
 
 ## Contributing
 
 Please see the [contributing guide](CONTRIBUTING.md) for information on adding icons or contributing code.
 
-For users familiar with the guidelines, check out [how to use icontool.py](/.github/icontool_guide.md) to speedup the process of adding icons.
+If you know how to create proper icons already, you can use [icontool.py](/.github/icontool_guide.md) to speed up the process of adding icons. Also see the [Figma contributing guide](https://www.figma.com/community/file/1227718471680779613).
 
 ## Requesting icons
 
 To request new icons, please use the [icon request form](https://forms.gle/Fx8vZAiWdW1Tyjo57).
 
-If you want to see the requested icons (for contributing purposes), you can view the form responses in our 
-[publicly available Google Sheets](https://docs.google.com/spreadsheets/d/1h3eiJnG2nEdR1DbvemaF1lYthHkzYbXvVFPP0TEEt5k/edit?usp=sharing) file.
-
-**Note that we currently don't show whether an icon request has been completed or not.**
-
-## License
-
-Lawnicons is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). See [LICENSE](LICENSE) for additional information.
-
-## Quick links
-
-- [Lawnchair GitHub page](https://github.com/LawnchairLauncher/lawnchair)
-- [Lawnicons Guidelines | Figma document](https://www.figma.com/community/file/1227718471680779613)
+You can also view the [raw requests](https://docs.google.com/spreadsheets/d/1h3eiJnG2nEdR1DbvemaF1lYthHkzYbXvVFPP0TEEt5k/edit?usp=sharing) or a [summary of the requests](https://docs.google.com/spreadsheets/d/10_o0DjhAGfP8ToDDNQ8M7eCZKtbzXLhIvxuF1XZEql0/edit).
```

Comment:
What's the difference between the two spreadsheets?

Reply:
Ahh I see now. In that case raw requests can get removed? The summary already has everything the raw has

Reply:
Only if nobody loses track of the original spreadsheet, as the second one is just importing cells from the original one
Should I?